bbc/simorgh
# WSTEAM1-844: Remove comma after Post Heading timestamp to reduce VoiceOver gestures

Resolves JIRA [WSTEAM1-844](https://jira.dev.bbc.co.uk/browse/WSTEAM1-844)

Overall changes
======
Removes visually hidden comma after Post Heading Timestamp to reduce VoiceOver gestures. (Since iOS 17+ does not support role="text" which is causing text splitting)

Note this provides benefits for screen reader users at the expense of how the page looks without CSS.

Code changes
======
- Remove visually hidden comma
- Update snapshots

Testing
======
1. Go to this testing link (http://tinyurl.com/4zhd6amv) on Safari on an iPhone on iOS 17 or later
2. Turn on Voiceover
3. Swipe through the Post Headings. 
4. If a breaking label is present, then the Timestamp, breaking label and headline should be announced with four gestures (Timestamp, breaking, comma, headline).
5. If there is no breaking label, then the Timestamp and the headline should be announced with two gestures. (Timestamp, headline).

There is no UX change.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
